### PR TITLE
Add lzma and zstd

### PR DESCRIPTION
**Pull request by goapunk**
We started conducting some tests on Tor's data usage (https://code.briarproject.org/akwizgran/briar/issues/45) and while at it I added support for lzma and zstd to see if it makes a difference. I thought this could be useful here too.
(In case you already got something prepared on your own feel free to close this MR :) )
Signed-off-by: goapunk <noobie@goapunks.net>

### CODE REVIEW

File: external/Makefile [hunk 1]
```diff
@@ -216,14 +268,13 @@ tor/Makefile: tor/configure
 				--disable-asciidoc \
 				--enable-static-libevent --with-libevent-dir=$(EXTERNAL_ROOT) \
 				--enable-static-openssl --with-openssl-dir=$(EXTERNAL_ROOT) \
-				--disable-linker-hardening --disable-gcc-hardening --disable-tool-name-check \
-				--disable-lzma --disable-zstd
+				--disable-linker-hardening --disable-gcc-hardening --disable-tool-name-check --disable-systemd
```
Comment (goapunk) [inline external/Makefile]: On Fedora I need to disable systemd support, otherwise it won't compile.

**Comment by n8fr8** [comment]
Fanastic, will review and merge. FYI, the plan is to break out the building of tor into its own library project and gradle.

**Comment by n8fr8** [comment]
Thanks again, and keep us posted on the data measurements!

**Comment by goapunk** [comment]
>FYI, the plan is to break out the building of tor into its own library project and gradle.

Great idea! Making the Tor build process reproducible is one of our upcoming tasks. I'll keep you posted about any positive results.
>Thanks again, and keep us posted on the data measurements!

Will do :)

**Pull request merged**